src/pynxtools/dataconverter/readers/utils.py

```diff
@@ -46,7 +46,7 @@ class FlattenSettings:
     dic: Mapping
     convert_dict: dict
     replace_nested: dict
-    parent_key: str = "/ENTRY[entry]"
+    parent_key: str = "/ENTRY"
```

Comment:
Thank you. I missed this value. Can you also change the `Defaults to "/ENTRY[entry]"` in the docstring?

Reply:
done